### PR TITLE
Update links and copyright year in documentation

### PR DESCRIPTION
**Pull request by petrkajzar**
- Remove two dead links.
- Move links from HTTP to HTTPS, as this should be standard.
- Update copyright year from 2015 to 2019.

### CODE REVIEW

File: data/pages/wiki/dokuwiki.txt [hunk 1]
```diff
@@ -48,17 +47,16 @@ All documentation and additional information besides the [[syntax|syntax descrip
...
   * [[doku>irc|Talk to other users in the IRC channel]]
   * [[https://github.com/splitbrain/dokuwiki/issues|Submit bugs and feature wishes]]
-  * [[http://www.wikimatrix.org/forum/viewforum.php?id=10|Share your experiences in the WikiMatrix forum]]
   * [[doku>thanks|Some humble thanks]]
 
 
 ===== Copyright =====
 
-2004-2015 (c) Andreas Gohr <andi@splitbrain.org>((Please do not contact me for help and support -- use the [[doku>mailinglist]] or [[http://forum.dokuwiki.org|forum]] instead)) and the DokuWiki Community
+2004-2019 (c) Andreas Gohr <andi@splitbrain.org>((Please do not contact me for help and support -- use the [[doku>mailinglist]] or [[https://forum.dokuwiki.org|forum]] instead)) and the DokuWiki Community
```
Comment (phy25) [inline data/pages/wiki/dokuwiki.txt]: I would prefer this to be 2020 and merge later.

Comment (petrkajzar) [reply data/pages/wiki/dokuwiki.txt]: OK, why not.

**Comment by Klap-in** [comment]
For me, one year ahead is fine (for just a month). I would not wait merging this change.

**Comment by phy25** [comment]
@Klap-in Good point! Just realized that North America is now in holiday season - kind of "next year is almost here".

**Pull request merged**